### ISSUE TITLE
MacOS 10.13 tgz is not supported

### ISSUE THREAD

**Issue by stefancrain** [shyiko/jabba#106]
Hi All. 

I'm seeing a failure in 10.13.1 Beta (17B35a)

```
jabba install adopt@1.8-0
Downloading adopt@1.8.162-00 (https://github.com/AdoptOpenJDK/openjdk8-releases/releases/download/jdk8u162-b00/OpenJDK8_x64_Mac_jdk8u162-b00.tar.gz)
76429409/76429409
tgz is not supported
```

**Comment by lreimer** [comment]
Same for me. Any known workarounds?

**Comment by shyiko** [comment]
Fixed in [0.9.1](https://github.com/shyiko/jabba/blob/master/CHANGELOG.md#091---2017-10-12), available immediately :balloon:. Thank you for the ticket!
